sklearn/model_selection/_validation.py

```diff
@@ -169,14 +169,14 @@ def cross_validate(
 
     scoring : str, callable, list, tuple, or dict, default=None
         Strategy to evaluate the performance of the cross-validated model on
-        the test set. If `None`, the
-        :ref:`default evaluation criterion <scoring_api_overview>` of the estimator
-        is used.
+        the test set.
```

Comment:
The term "test set" here makes me feel as if the user had to provided the test set coming from a train-test split. I guess "validation set" or just "left out data" would be more appropriate, but maybe it's only me. Alternatively, we could use a wording similar to
> Strategy to evaluate the performance of the model across cross-validation splits.

If you agree, this comment also applies to `cross_val_score` and maybe slightly adapt it for `permutation_test_score`.

Reply:
I like:

> Strategy to evaluate the performance of the model across cross-validation splits.

I've altered to use `estimator`, to reference the `estimator` parameter:

Strategy to evaluate the performance of the `estimator` across cross-validation splits.

Am happy to change though.

For `permutation_test_score` I have used "validation set", as we use CV to generate the null hypothesis distribution as well, so I thought "across cross-validation splits" was not as fitting. Happy to change.